cmd/gosky/util/util.go

```diff
@@ -152,6 +152,8 @@ func SetupDatabase(dburl string) (*gorm.DB, error) {
 	var dial gorm.Dialector
 	// NOTE(bnewbold): might also handle file:// as sqlite, but let's keep it
 	// explicit for now
+
+	openConns := 99
```

Comment:
I think this is pretty high for postgresql by default? I know there have been improvements in recent years, but I think the postgresql design philosophy is to keep connections to a couple dozen at max and go with `pg_bounce` if needed. Probably fine on our beefy deployed instances. I would have gone with 20 or 30.

Reply:
this is really just copy/paste from my estuary code, picked it there on recommendation of some perf guides somewhere, totally not opinionated about it